### DISCUSSION TITLE
Thank you

### DISCUSSION THREAD

**Discussion by andrewmcwatters** [r-lyeh/FWK#36] [Random]
I noticed `Planimeter/lgf`, which I author, mentioned here in your repository. I also greatly enjoy C game engines. Thank you for your contributions to this space. FWK is very impressive.

-Andrew

**Comment by r-lyeh** [comment]
Oh nice! LGF has always been a major inspiration to me. In fact, LGF was one of the things that motivated me to rewrite a previous version of the engine into something much smaller. Also, one of the reasons I provide dll+luajit support out of the box is just because I really liked the way LGF is distributed. So classy. 

Glad that you like the project! Keep up the good work on LGF as well!
